content/en/docs/ops/common-problems/network-issues/index.md

```diff
@@ -233,7 +233,235 @@ server {
 }
 {{< /text >}}
 
-## 404 errors occur when multiple gateways configured with same TLS certificate
+## TLS configuration mistakes
+
+Many traffic managment problems
+are caused by incorrect [TLS configuration](/docs/ops/configuration/traffic-management/tls-configuration/).
+The following sections describe some of the most common misconfigurations.
+
+### Sending HTTPS to an HTTP port
+
+If your application sends an HTTPS request to a service declared to be HTTP,
+the Envoy sidecar will attempt to parse the request as HTTP while forwarding the request,
+which will fail because the HTTP is unexpectedly encrypted.
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1beta1
+kind: ServiceEntry
+metadata:
+  name: httpbin
+spec:
+  hosts:
+  - httpbin.org
+  ports:
+  - number: 443
+    name: http
+    protocol: HTTP
+  resolution: DNS
+{{< /text >}}
+
+Although the above configuration may be correct if you are intentionally sending plaintext on port 443 (e.g., `curl http://httpbin.org:443`),
+generally port 443 is dedicated for HTTPS traffic.
+
+Sending an HTTPS request like `curl https://httpbin.org`, which defaults to port 443, will result in an error something like
```

Comment:
nit: `error something like ...` seems off. Maybe should be "error, something like ..."?